layouts/request-and-response.html

```diff
@@ -0,0 +1,5 @@
+<h1 id="request-and-response">Request & Response</h1>
+
+<p>The Request & Response resources track information conveyance between one or more entities.</p>
```

Comment:
Is this description okay? (I know this is base stuff that I added, it's just this wording was the wording I was must unsure about & it's not green on my review).

Reply:
@ebaranowski any changes needed?

Reply:
Ask the SD group?

Reply:
Yeah have to get this from the SD group

Reply:
Will discuss this with the SDs in a meeting tomorrow, so resolving.